### PR TITLE
kubelet/sysctl: update log level

### PR DESCRIPTION
**Pull request by cyclinder**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

update log level from error to info, which is fix for https://github.com/kubernetes/kubernetes/pull/118846#discussion_r1357528074.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/kubelet/sysctl/safe_sysctls.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ func getSafeSysctlAllowlist(getVersion func() (*version.Version, error)) []strin
 		if kernelVersion != nil && kernelVersion.AtLeast(version.MustParseGeneric(sc.kernel)) {
 			safeSysctlAllowlist = append(safeSysctlAllowlist, sc.name)
 		} else {
-			klog.ErrorS(nil, "kernel version is too small, dropping the sysctl from safe sysctl list", "kernelVersion", kernelVersion, "sysctl", sc.name)
+			klog.InfoS("kernel version is too old, dropping the sysctl from safe sysctl list", "kernelVersion", kernelVersion, "sysctl", sc.name)
```
Comment (bart0sh) [inline pkg/kubelet/sysctl/safe_sysctls.go:85]: This message would appear in the log if kernelVersion is nil, which makes it not entirely correct. I'd suggest to check if it really can be nil and either change the condition above or the message.

Comment (thockin) [reply pkg/kubelet/sysctl/safe_sysctls.go:85]: if `getVersion()` returned `nil` and not an error, it's broken.  Really, we don't need this nil check here (or we should check it above alongside checking err)

**Comment by thockin** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**